.jenkins/pace_physics_cache_setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [[ "${NODE_NAME}" == *"daint"* ]] ; then source ~/.bashrc ; fi
+set -e
+export LONG_EXECUTION=1
+.jenkins/jenkins.sh run_physics_regression_tests ${backend} ${experiment}
+cd buildenv && git reset --hard && cd ..
```

Comment:
Why is this needed, but not for the rest of the repo? Is it because jenkins.sh leaves buildenv in a dirty state? If so, we may want to fix that (but as a todo Jira task, not right now).

Reply:
Yes, what is strange is that if instead of calling `pace_physics_cache_setup`, we have the same commands in he jenkins exec shell, then buildenv gets reset. I'll put a note for this.